sources/packages/backend/apps/db-migrations/src/sql/Queue/Add-student-loan-balances-queue.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO
+  sims.queue_configurations(queue_name, queue_configuration)
+VALUES
+  (
+    'student-loan-balances',
```

Comment:
All the integration scheduler queues are suffixed with `-integration`. We should follow the same here.

Also this is a scheduler for PT. Queue name must indicate it.